terraspace/app/stacks/bucket-mirror/main.tf

```diff
@@ -71,4 +71,9 @@ resource "aws_instance" "bucket_mirror_runner" {
   volume_tags = {
     Name = "bucket-mirror"
   }
+
+  metadata_options {
+    http_endpoint = "enabled"
+    http_tokens   = "required"
+  }
```

Comment:
Just curious, is this runner script using IMDSv2?

Reply:
I'll tell you in slack for security reasons :smile: